src/color.js

```diff
@@ -64,6 +64,11 @@ export default class Color {
 
 		this.coords = coords ? coords.slice() : [0, 0, 0];
 
+		// Ensure alpha is a number
+		if (!Number.isFinite(alpha)) {
+			alpha = 1;
+		}
+
 		// Clamp alpha to [0, 1]
 		this.alpha = alpha > 1 ? 1 : (alpha < 0 ? 0 : alpha);
```

Comment:
Would something like this work?
```suggestion
		// Clamp alpha to [0, 1]
		this.alpha = alpha < 1 && Number.isFinite(alpha) ? (alpha < 0 ? 0 : alpha) : 1;
```

Or even:
```suggestion
		// Clamp alpha to [0, 1]
		this.alpha = Math.min(Math.max(0, alpha), 1) : 1;
```

Or (this should work even for non-numerical values):

Or even:
```suggestion
		// Clamp alpha to [0, 1]
		this.alpha = 0 <= alpha && alpha <= 1? alpha : 1;
```

Reply:
I preferred the last suggestion

Reply:
I just realized this had an issue: it does not actually clamp, if you pass an alpha of -1, it will also become 1 (which is both unintuitive and a breaking change).

How about:

```
// Clamp alpha to [0, 1]
this.alpha = Number.isNaN(alpha)? 1 : Math.max(0, Math.min(alpha, 1));
```